testnet/scripts/run_nomos_bot.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+echo "I am a container ${HOSTNAME} bot"
+
+exec /usr/bin/nomos-cli disseminate /etc/nomos/libp2p_config.yaml --with-metrics
```

Comment:
Asking just to understand the script. Do we need to execute this dissemination here before starting the chat bot?

Reply:
I knew I've missed something, thanks for catching. Maybe later we'll constantly push file to network, but at the moment this is just a leftover, will remove.

Reply:
Aha, I understand, that's why the name of this script is `run_nomos_bot.sh`, not `run_nomos_chatbot.sh`. It makes sense. Thank you!